Synthesis failed for nodejs-appengine-admin

Hello! Autosynth couldn't regenerate nodejs-appengine-admin. :broken_heart:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the nodejs-appengine-admin API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
 Running: git log -1 --pretty=%at f4c3a57aeddad8ad720e075a9e8342f1f7b9b766
2021-04-16 03:30:50,673 autosynth [DEBUG] > Running: git log -1 --pretty=%at 705743e66f5c0b24a95f7f30619c9d3ef747b317
2021-04-16 03:30:50,676 autosynth [DEBUG] > Running: git log -1 --pretty=%at 0f3469e88a4be9af4d9b1c4b6cf6fd3f8ac15588
2021-04-16 03:30:50,679 autosynth [DEBUG] > Running: git log -1 --pretty=%at fbc7b766ef43c75293b26159ecaf66cb3fb23770
2021-04-16 03:30:50,682 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6444141a0ab125ff5579e0b697b41ccc500cbda7
2021-04-16 03:30:50,685 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5b0e1592dd7d70b485e157ea4b3eb1704ecbd015
2021-04-16 03:30:50,688 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1f5e6bc8dc8e3661ee550905fc070e55e1b6cea1
2021-04-16 03:30:50,692 autosynth [DEBUG] > Running: git log -1 --pretty=%at d9ddac83a22a600dd33854c9d835a4fe52284207
2021-04-16 03:30:50,695 autosynth [DEBUG] > Running: git log -1 --pretty=%at 0a071b3460344886297a304253bf924aa68ddb7e
2021-04-16 03:30:50,698 autosynth [DEBUG] > Running: git log -1 --pretty=%at 082e1ca0863b13ada8594fe91845380765da5b70
2021-04-16 03:30:50,702 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8285c2b4cdbc3771d031ad91e1c4ec9e55fff45d
2021-04-16 03:30:50,705 autosynth [DEBUG] > Running: git log -1 --pretty=%at 89a8b2e692f7208103647750496610832abebdd0
2021-04-16 03:30:50,708 autosynth [DEBUG] > Running: git log -1 --pretty=%at 721339ab60a6eb63b889978b3d9b295dcb3be370
2021-04-16 03:30:50,711 autosynth [DEBUG] > Running: git log -1 --pretty=%at 14111b3ce3334271d8a7a2bc6c108cfccca9943e
2021-04-16 03:30:50,714 autosynth [DEBUG] > Running: git log -1 --pretty=%at 043cc620d6a6111816d9e09f2a97208565fde958
2021-04-16 03:30:50,717 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1657dd8c79a82b3fe9dde3ff6e0eda3d47fb117f
2021-04-16 03:30:50,720 autosynth [DEBUG] > Running: git log -1 --pretty=%at 898b38a6f4fab89a76dfb152480bb034a781331b
2021-04-16 03:30:50,724 autosynth [DEBUG] > Running: git checkout bd47983afdd6f80df84a0310f523ebe01ad3d312
Note: checking out 'bd47983afdd6f80df84a0310f523ebe01ad3d312'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at bd47983 chore(deps): update dependency @types/sinon to v10 (#26)
2021-04-16 03:30:50,754 autosynth [DEBUG] > Running: git checkout 898b38a6f4fab89a76dfb152480bb034a781331b
Note: checking out '898b38a6f4fab89a76dfb152480bb034a781331b'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 898b38a feat: fetch for default branch  (#1031)
2021-04-16 03:30:50,762 autosynth [DEBUG] > Running: git checkout b1c23ed98caaae6b5d0cf75649afc803bda141f3
Note: checking out 'b1c23ed98caaae6b5d0cf75649afc803bda141f3'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b1c23ed9 feat!: Refactored Elliptic Key curves to provide additional options, the NamedCurve enum is replaced by the EcKeyType message.
2021-04-16 03:30:50,777 autosynth [DEBUG] > Running: git branch -f autosynth-156
2021-04-16 03:30:50,781 autosynth [DEBUG] > Running: git checkout autosynth-156
Switched to branch 'autosynth-156'
2021-04-16 03:30:50,785 autosynth [INFO] > Running synthtool
2021-04-16 03:30:50,786 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-04-16 03:30:50,786 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/nodejs-appengine-admin/156/sponge_log.log
2021-04-16 03:30:50,787 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-04-16 03:30:51,021 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-appengine-admin/synth.py.
On branch autosynth-156
nothing to commit, working tree clean
2021-04-16 03:30:51,153 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2021-04-16 03:30:51,163 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-04-16 03:30:51,164 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-04-16 03:30:51,168 synthtool [DEBUG] > Which version of bazel will I run?
DEBUG:synthtool:Which version of bazel will I run?
bazel 3.0.0
2021-04-16 03:30:51,171 synthtool [DEBUG] > Generating code for: //google/appengine/v1:appengine-v1-nodejs.
DEBUG:synthtool:Generating code for: //google/appengine/v1:appengine-v1-nodejs.
2021-04-16 03:30:57,092 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpd9kbdsp0.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpd9kbdsp0.
2021-04-16 03:30:57,114 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-04-16 03:30:57,131 synthtool [INFO] > successfully generate `src/index.ts`
INFO:synthtool:successfully generate `src/index.ts`
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-appengine-admin/synth.py", line 38, in <module>
    source_location='build/src', versions=["v1"], default_version="v1")
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 301, in node_library
    return self._generic_library("node_library", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 65, in _generic_library
    kwargs["metadata"]["repository"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 374, in _get_default_branch_name
    github_req.raise_for_status()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: rate limit exceeded for url: https://api.github.com/repos/googleapis/nodejs-appengine
2021-04-16 03:30:57,245 autosynth [ERROR] > Synthesis failed
2021-04-16 03:30:57,245 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bd47983 chore(deps): update dependency @types/sinon to v10 (#26)
2021-04-16 03:30:57,255 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-04-16 03:30:57,265 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 356, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 336, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0f7d18dd-0fed-4284-8748-9a058bdad02f/targets/github%2Fsynthtool;config=default/tests;query=nodejs-appengine-admin;failed=false).

Autosynth is still having trouble generating nodejs-appengine-admin. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the nodejs-appengine-admin API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
af66cb3fb23770
2021-04-17 03:29:39,561 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6444141a0ab125ff5579e0b697b41ccc500cbda7
2021-04-17 03:29:39,564 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5b0e1592dd7d70b485e157ea4b3eb1704ecbd015
2021-04-17 03:29:39,567 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1f5e6bc8dc8e3661ee550905fc070e55e1b6cea1
2021-04-17 03:29:39,570 autosynth [DEBUG] > Running: git log -1 --pretty=%at d9ddac83a22a600dd33854c9d835a4fe52284207
2021-04-17 03:29:39,573 autosynth [DEBUG] > Running: git log -1 --pretty=%at 0a071b3460344886297a304253bf924aa68ddb7e
2021-04-17 03:29:39,576 autosynth [DEBUG] > Running: git log -1 --pretty=%at 082e1ca0863b13ada8594fe91845380765da5b70
2021-04-17 03:29:39,579 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8285c2b4cdbc3771d031ad91e1c4ec9e55fff45d
2021-04-17 03:29:39,582 autosynth [DEBUG] > Running: git log -1 --pretty=%at 89a8b2e692f7208103647750496610832abebdd0
2021-04-17 03:29:39,585 autosynth [DEBUG] > Running: git log -1 --pretty=%at 721339ab60a6eb63b889978b3d9b295dcb3be370
2021-04-17 03:29:39,588 autosynth [DEBUG] > Running: git log -1 --pretty=%at 14111b3ce3334271d8a7a2bc6c108cfccca9943e
2021-04-17 03:29:39,591 autosynth [DEBUG] > Running: git log -1 --pretty=%at 043cc620d6a6111816d9e09f2a97208565fde958
2021-04-17 03:29:39,594 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1657dd8c79a82b3fe9dde3ff6e0eda3d47fb117f
2021-04-17 03:29:39,597 autosynth [DEBUG] > Running: git log -1 --pretty=%at 898b38a6f4fab89a76dfb152480bb034a781331b
2021-04-17 03:29:39,600 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6e8322421485d0937cb4758db812d8fbca0bbedd
2021-04-17 03:29:39,603 autosynth [DEBUG] > Running: git log -1 --pretty=%at b33b0e2056a85fc2264b294f2cf47dcd45e95186
2021-04-17 03:29:39,606 autosynth [DEBUG] > Running: git log -1 --pretty=%at c6706ee5d693e9ae5967614170732646590d8374
2021-04-17 03:29:39,609 autosynth [DEBUG] > Running: git log -1 --pretty=%at ff6b759fcb58f216f62b48d5be0bb85fafbc3bb2
2021-04-17 03:29:39,613 autosynth [DEBUG] > Running: git checkout bd47983afdd6f80df84a0310f523ebe01ad3d312
Note: checking out 'bd47983afdd6f80df84a0310f523ebe01ad3d312'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at bd47983 chore(deps): update dependency @types/sinon to v10 (#26)
2021-04-17 03:29:39,642 autosynth [DEBUG] > Running: git checkout ff6b759fcb58f216f62b48d5be0bb85fafbc3bb2
Note: checking out 'ff6b759fcb58f216f62b48d5be0bb85fafbc3bb2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at ff6b759 chore: Fix python owlbot post processor image cloudbuild.yaml (#1054)
2021-04-17 03:29:39,650 autosynth [DEBUG] > Running: git checkout a8cd11e2c420a194348839c6490a8a1bef2835d3
Note: checking out 'a8cd11e2c420a194348839c6490a8a1bef2835d3'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a8cd11e2 feat: publish serviceusage/v1beta1 API
2021-04-17 03:29:39,663 autosynth [DEBUG] > Running: git branch -f autosynth-163
2021-04-17 03:29:39,667 autosynth [DEBUG] > Running: git checkout autosynth-163
Switched to branch 'autosynth-163'
2021-04-17 03:29:39,671 autosynth [INFO] > Running synthtool
2021-04-17 03:29:39,671 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-04-17 03:29:39,671 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/nodejs-appengine-admin/163/sponge_log.log
2021-04-17 03:29:39,673 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-04-17 03:29:39,897 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-appengine-admin/synth.py.
On branch autosynth-163
nothing to commit, working tree clean
2021-04-17 03:29:40,025 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2021-04-17 03:29:40,035 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-04-17 03:29:40,036 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-04-17 03:29:40,040 synthtool [DEBUG] > Which version of bazel will I run?
DEBUG:synthtool:Which version of bazel will I run?
bazel 3.0.0
2021-04-17 03:29:40,044 synthtool [DEBUG] > Generating code for: //google/appengine/v1:appengine-v1-nodejs.
DEBUG:synthtool:Generating code for: //google/appengine/v1:appengine-v1-nodejs.
2021-04-17 03:29:45,518 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpml8_d0ap.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpml8_d0ap.
2021-04-17 03:29:45,541 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-04-17 03:29:45,557 synthtool [INFO] > successfully generate `src/index.ts`
INFO:synthtool:successfully generate `src/index.ts`
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-appengine-admin/synth.py", line 38, in <module>
    source_location='build/src', versions=["v1"], default_version="v1")
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 301, in node_library
    return self._generic_library("node_library", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 65, in _generic_library
    kwargs["metadata"]["repository"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 374, in _get_default_branch_name
    github_req.raise_for_status()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: rate limit exceeded for url: https://api.github.com/repos/googleapis/nodejs-appengine
2021-04-17 03:29:45,653 autosynth [ERROR] > Synthesis failed
2021-04-17 03:29:45,653 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bd47983 chore(deps): update dependency @types/sinon to v10 (#26)
2021-04-17 03:29:45,664 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-04-17 03:29:45,674 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 356, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 336, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/2ea6930c-6999-4b7b-87b2-ebae50f55cbb/targets/github%2Fsynthtool;config=default/tests;query=nodejs-appengine-admin;failed=false).

Autosynth is still having trouble generating nodejs-appengine-admin. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the nodejs-appengine-admin API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
93b26159ecaf66cb3fb23770
2021-04-18 04:18:57,298 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6444141a0ab125ff5579e0b697b41ccc500cbda7
2021-04-18 04:18:57,301 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5b0e1592dd7d70b485e157ea4b3eb1704ecbd015
2021-04-18 04:18:57,304 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1f5e6bc8dc8e3661ee550905fc070e55e1b6cea1
2021-04-18 04:18:57,307 autosynth [DEBUG] > Running: git log -1 --pretty=%at d9ddac83a22a600dd33854c9d835a4fe52284207
2021-04-18 04:18:57,310 autosynth [DEBUG] > Running: git log -1 --pretty=%at 0a071b3460344886297a304253bf924aa68ddb7e
2021-04-18 04:18:57,313 autosynth [DEBUG] > Running: git log -1 --pretty=%at 082e1ca0863b13ada8594fe91845380765da5b70
2021-04-18 04:18:57,316 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8285c2b4cdbc3771d031ad91e1c4ec9e55fff45d
2021-04-18 04:18:57,320 autosynth [DEBUG] > Running: git log -1 --pretty=%at 89a8b2e692f7208103647750496610832abebdd0
2021-04-18 04:18:57,323 autosynth [DEBUG] > Running: git log -1 --pretty=%at 721339ab60a6eb63b889978b3d9b295dcb3be370
2021-04-18 04:18:57,326 autosynth [DEBUG] > Running: git log -1 --pretty=%at 14111b3ce3334271d8a7a2bc6c108cfccca9943e
2021-04-18 04:18:57,329 autosynth [DEBUG] > Running: git log -1 --pretty=%at 043cc620d6a6111816d9e09f2a97208565fde958
2021-04-18 04:18:57,332 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1657dd8c79a82b3fe9dde3ff6e0eda3d47fb117f
2021-04-18 04:18:57,335 autosynth [DEBUG] > Running: git log -1 --pretty=%at 898b38a6f4fab89a76dfb152480bb034a781331b
2021-04-18 04:18:57,338 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6e8322421485d0937cb4758db812d8fbca0bbedd
2021-04-18 04:18:57,341 autosynth [DEBUG] > Running: git log -1 --pretty=%at b33b0e2056a85fc2264b294f2cf47dcd45e95186
2021-04-18 04:18:57,344 autosynth [DEBUG] > Running: git log -1 --pretty=%at c6706ee5d693e9ae5967614170732646590d8374
2021-04-18 04:18:57,347 autosynth [DEBUG] > Running: git log -1 --pretty=%at ff6b759fcb58f216f62b48d5be0bb85fafbc3bb2
2021-04-18 04:18:57,351 autosynth [DEBUG] > Running: git checkout bd47983afdd6f80df84a0310f523ebe01ad3d312
Note: checking out 'bd47983afdd6f80df84a0310f523ebe01ad3d312'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at bd47983 chore(deps): update dependency @types/sinon to v10 (#26)
2021-04-18 04:18:57,355 autosynth [DEBUG] > Running: git checkout ff6b759fcb58f216f62b48d5be0bb85fafbc3bb2
Note: checking out 'ff6b759fcb58f216f62b48d5be0bb85fafbc3bb2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at ff6b759 chore: Fix python owlbot post processor image cloudbuild.yaml (#1054)
2021-04-18 04:18:57,361 autosynth [DEBUG] > Running: git checkout a8cd11e2c420a194348839c6490a8a1bef2835d3
Note: checking out 'a8cd11e2c420a194348839c6490a8a1bef2835d3'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a8cd11e2 feat: publish serviceusage/v1beta1 API
2021-04-18 04:18:57,403 autosynth [DEBUG] > Running: git branch -f autosynth-163
2021-04-18 04:18:57,407 autosynth [DEBUG] > Running: git checkout autosynth-163
Switched to branch 'autosynth-163'
2021-04-18 04:18:57,412 autosynth [INFO] > Running synthtool
2021-04-18 04:18:57,412 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-04-18 04:18:57,412 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/nodejs-appengine-admin/163/sponge_log.log
2021-04-18 04:18:57,414 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-04-18 04:18:57,646 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-appengine-admin/synth.py.
On branch autosynth-163
nothing to commit, working tree clean
2021-04-18 04:18:57,778 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2021-04-18 04:18:57,790 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-04-18 04:18:57,791 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-04-18 04:18:57,796 synthtool [DEBUG] > Which version of bazel will I run?
DEBUG:synthtool:Which version of bazel will I run?
bazel 3.0.0
2021-04-18 04:18:57,799 synthtool [DEBUG] > Generating code for: //google/appengine/v1:appengine-v1-nodejs.
DEBUG:synthtool:Generating code for: //google/appengine/v1:appengine-v1-nodejs.
2021-04-18 04:19:24,284 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpr5pi0b1t.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpr5pi0b1t.
2021-04-18 04:19:24,304 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-04-18 04:19:24,321 synthtool [INFO] > successfully generate `src/index.ts`
INFO:synthtool:successfully generate `src/index.ts`
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-appengine-admin/synth.py", line 38, in <module>
    source_location='build/src', versions=["v1"], default_version="v1")
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 301, in node_library
    return self._generic_library("node_library", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 65, in _generic_library
    kwargs["metadata"]["repository"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 374, in _get_default_branch_name
    github_req.raise_for_status()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.github.com/repos/googleapis/nodejs-appengine
2021-04-18 04:19:24,501 autosynth [ERROR] > Synthesis failed
2021-04-18 04:19:24,502 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bd47983 chore(deps): update dependency @types/sinon to v10 (#26)
2021-04-18 04:19:24,512 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-04-18 04:19:24,522 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 356, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 336, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/bcc0b7f5-e83d-4774-b6e2-75a108517b3a/targets/github%2Fsynthtool;config=default/tests;query=nodejs-appengine-admin;failed=false).

Autosynth is still having trouble generating nodejs-appengine-admin. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the nodejs-appengine-admin API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
93b26159ecaf66cb3fb23770
2021-04-19 03:32:42,889 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6444141a0ab125ff5579e0b697b41ccc500cbda7
2021-04-19 03:32:42,892 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5b0e1592dd7d70b485e157ea4b3eb1704ecbd015
2021-04-19 03:32:42,896 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1f5e6bc8dc8e3661ee550905fc070e55e1b6cea1
2021-04-19 03:32:42,899 autosynth [DEBUG] > Running: git log -1 --pretty=%at d9ddac83a22a600dd33854c9d835a4fe52284207
2021-04-19 03:32:42,902 autosynth [DEBUG] > Running: git log -1 --pretty=%at 0a071b3460344886297a304253bf924aa68ddb7e
2021-04-19 03:32:42,905 autosynth [DEBUG] > Running: git log -1 --pretty=%at 082e1ca0863b13ada8594fe91845380765da5b70
2021-04-19 03:32:42,908 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8285c2b4cdbc3771d031ad91e1c4ec9e55fff45d
2021-04-19 03:32:42,912 autosynth [DEBUG] > Running: git log -1 --pretty=%at 89a8b2e692f7208103647750496610832abebdd0
2021-04-19 03:32:42,915 autosynth [DEBUG] > Running: git log -1 --pretty=%at 721339ab60a6eb63b889978b3d9b295dcb3be370
2021-04-19 03:32:42,918 autosynth [DEBUG] > Running: git log -1 --pretty=%at 14111b3ce3334271d8a7a2bc6c108cfccca9943e
2021-04-19 03:32:42,921 autosynth [DEBUG] > Running: git log -1 --pretty=%at 043cc620d6a6111816d9e09f2a97208565fde958
2021-04-19 03:32:42,924 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1657dd8c79a82b3fe9dde3ff6e0eda3d47fb117f
2021-04-19 03:32:42,927 autosynth [DEBUG] > Running: git log -1 --pretty=%at 898b38a6f4fab89a76dfb152480bb034a781331b
2021-04-19 03:32:42,931 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6e8322421485d0937cb4758db812d8fbca0bbedd
2021-04-19 03:32:42,934 autosynth [DEBUG] > Running: git log -1 --pretty=%at b33b0e2056a85fc2264b294f2cf47dcd45e95186
2021-04-19 03:32:42,937 autosynth [DEBUG] > Running: git log -1 --pretty=%at c6706ee5d693e9ae5967614170732646590d8374
2021-04-19 03:32:42,940 autosynth [DEBUG] > Running: git log -1 --pretty=%at ff6b759fcb58f216f62b48d5be0bb85fafbc3bb2
2021-04-19 03:32:42,944 autosynth [DEBUG] > Running: git checkout bd47983afdd6f80df84a0310f523ebe01ad3d312
Note: checking out 'bd47983afdd6f80df84a0310f523ebe01ad3d312'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at bd47983 chore(deps): update dependency @types/sinon to v10 (#26)
2021-04-19 03:32:42,974 autosynth [DEBUG] > Running: git checkout ff6b759fcb58f216f62b48d5be0bb85fafbc3bb2
Note: checking out 'ff6b759fcb58f216f62b48d5be0bb85fafbc3bb2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at ff6b759 chore: Fix python owlbot post processor image cloudbuild.yaml (#1054)
2021-04-19 03:32:42,983 autosynth [DEBUG] > Running: git checkout a8cd11e2c420a194348839c6490a8a1bef2835d3
Note: checking out 'a8cd11e2c420a194348839c6490a8a1bef2835d3'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a8cd11e2 feat: publish serviceusage/v1beta1 API
2021-04-19 03:32:43,026 autosynth [DEBUG] > Running: git branch -f autosynth-163
2021-04-19 03:32:43,030 autosynth [DEBUG] > Running: git checkout autosynth-163
Switched to branch 'autosynth-163'
2021-04-19 03:32:43,035 autosynth [INFO] > Running synthtool
2021-04-19 03:32:43,035 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-04-19 03:32:43,035 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/nodejs-appengine-admin/163/sponge_log.log
2021-04-19 03:32:43,038 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-04-19 03:32:43,278 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-appengine-admin/synth.py.
On branch autosynth-163
nothing to commit, working tree clean
2021-04-19 03:32:43,414 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2021-04-19 03:32:43,425 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2021-04-19 03:32:43,426 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-04-19 03:32:43,430 synthtool [DEBUG] > Which version of bazel will I run?
DEBUG:synthtool:Which version of bazel will I run?
bazel 3.0.0
2021-04-19 03:32:43,434 synthtool [DEBUG] > Generating code for: //google/appengine/v1:appengine-v1-nodejs.
DEBUG:synthtool:Generating code for: //google/appengine/v1:appengine-v1-nodejs.
2021-04-19 03:33:12,547 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpv7y258fi.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpv7y258fi.
2021-04-19 03:33:12,582 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-04-19 03:33:12,607 synthtool [INFO] > successfully generate `src/index.ts`
INFO:synthtool:successfully generate `src/index.ts`
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-appengine-admin/synth.py", line 38, in <module>
    source_location='build/src', versions=["v1"], default_version="v1")
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 301, in node_library
    return self._generic_library("node_library", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 65, in _generic_library
    kwargs["metadata"]["repository"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/common.py", line 374, in _get_default_branch_name
    github_req.raise_for_status()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.github.com/repos/googleapis/nodejs-appengine
2021-04-19 03:33:12,816 autosynth [ERROR] > Synthesis failed
2021-04-19 03:33:12,816 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bd47983 chore(deps): update dependency @types/sinon to v10 (#26)
2021-04-19 03:33:12,830 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-04-19 03:33:12,844 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 356, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 336, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b6bb639a-b591-4a9d-9ae5-e9a536767a22/targets/github%2Fsynthtool;config=default/tests;query=nodejs-appengine-admin;failed=false).

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "@google-cloud/appengine-admin",
   "version": "0.2.0",
   "description": "Appengine client for Node.js",
-  "repository": "googleapis/nodejs-appengine",
+  "repository": "googleapis/nodejs-appengine-admin",
   "license": "Apache-2.0",
   "author": "Google LLC",
   "main": "build/src/index.js",

